### PR TITLE
OCM-6309 | ci: Setup day2 test cases

### PR DESCRIPTION
**Pull request by radtriste**
Those are not activated yet. Just migrated, following the [new naming convention](https://issues.redhat.com/browse/OCM-6301) 

- 37 test cases imported as stated and verified in [OCM-6305](https://issues.redhat.com/browse/OCM-6305)
  - Those are exluded for now
  - Other PRs will come to activate them by batch(es)
- vendor may be reduced later after other migration efforts and move from openshift-rosa-cli repository to ocm-common

<details>
<summary>$ ginkgo run -v --label-filter '(day1-post,day2)' --dry-run tests/e2e/</summary>
Will run 37 of 38 specs

[BeforeSuite] 
${ROSA_PATH}/tests/e2e/e2e_suite_test.go:19
[BeforeSuite] PASSED [0.000 seconds]

Region can list regions - [id:55729] [day2, feature-region, High, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_region.go:35
• [0.000 seconds]

Edit cluster -  can check the description of the cluster - [id:34102] [day2, feature-cluster, Medium, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_cluster.go:49
• [0.000 seconds]

Edit cluster -  can restrict master API endpoint to direct, private connectivity or not - [id:38850] [day2, feature-cluster, High, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_cluster.go:106
• [0.000 seconds]

Edit cluster -  can disable workload monitoring on/off - [id:45159] [day2, feature-cluster, High, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_cluster.go:168
• [0.000 seconds]

Edit cluster -  can validate for deletion of upgrade policy of rosa cluster - [id:38787] [day2, feature-cluster, Medium, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_cluster.go:216
• [0.000 seconds]

Edit cluster -  can allow sts cluster installation with compatible policies - [id:45161] [day2, feature-cluster, High, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_cluster.go:241
• [0.000 seconds]

S

Network verifier can verify network - [id:64917] [day2, feature-networkverifier, High, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_network_verifier.go:44
• [0.000 seconds]

Edit IDP can create/describe/delete admin user - [id:35878] [day2, feature-idp, Critical, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_idp.go:55
• [0.000 seconds]

Edit IDP can create/List/Delete IDPs for rosa clusters - [id:35896] [day2, feature-idp, Critical, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_idp.go:138
• [0.000 seconds]

Edit IDP can create/delete the HTPasswd IDPs - [id:49137] [day2, feature-idp, Critical, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_idp.go:275
• [0.000 seconds]

Validate user try to create cluster with invalid usernames, passwords or unsupported configurations - [id:66362] [day2, feature-user, Critical, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_user.go:144
• [0.000 seconds]

Edit IAM can create/list/delete account-roles - [id:43070] [day2, feature-roles, High, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_iam_roles.go:62
• [0.000 seconds]

Edit IAM can create operator-roles prior to cluster creation - [id:60971] [day2, feature-roles, High, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_iam_roles.go:239
• [0.000 seconds]

Edit IAM can validate when user create operator-roles to cluster - [id:43051] [day2, feature-roles, High, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_iam_roles.go:511
• [0.000 seconds]

Edit IAM can validate that upgrade account-roles with the managed policies should be forbidden - [id:57441] [day2, feature-roles, High, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_iam_roles.go:553
• [0.000 seconds]

Edit IAM can delete account-roles with --hosted-cp and --classic - [id:62083] [day2, feature-roles, High, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_iam_roles.go:615
• [0.000 seconds]

Edit IAM can validate create/link/unlink user-role - [id:52580] [day2, feature-roles, High, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_iam_roles.go:672
• [0.000 seconds]

Edit IAM can create/delete/unlink/link ocm-roles in auto mode - [id:46187] [day2, feature-roles, High, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_iam_roles.go:782
• [0.000 seconds]

Edit IAM can create/link/unlink/delete user-role in auto mode - [id:52419] [day2, feature-roles, High, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_iam_roles.go:918
• [0.000 seconds]

Edit kubeletconfig can create podPidLimit via rosacli will work well - [id:68828] [day2, feature-kubeletconfig, Critical, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_kubelet_config.go:47
• [0.000 seconds]

Edit kubeletconfig can update podPidLimit via rosacli will work well - [id:68835] [day2, feature-kubeletconfig, Critical, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_kubelet_config.go:88
• [0.000 seconds]

Edit kubeletconfig can delete podPidLimit via rosacli will work well - [id:68836] [day2, feature-kubeletconfig, Critical, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_kubelet_config.go:142
• [0.000 seconds]

Edit OIDC config can create/list/delete BYO oidc config in auto mode - [id:57570] [day2, feature-oidcconfig, High, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_oidc_config.go:54
• [0.000 seconds]

Verify the creation of rosa cluster with volume size will work - [id:66359] [day1-post, Critical, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_day1_post.go:52
• [0.000 seconds]

Verify the creation of ROSA cluster with default-mp-labels option will succeed - [id:57056] [day1-post, Critical, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_day1_post.go:95
• [0.000 seconds]

Verify the windows certificates expiration - [id:64040] [day1-post, Medium, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_day1_post.go:131
• [0.000 seconds]

Edit default ingress can update on rosa HCP cluster - [id:63323] [day2, feature-ingress, Critical, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_ingress.go:43
• [0.000 seconds]

OIDC provider validate when user create oidc-provider to cluster - [id:43046] [day2, feature-oidcprovider, High, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_oidc_provider.go:42
• [0.000 seconds]

Edit machinepool can create machinepool with volume size set - [id:66872] [day2, feature-machinepool, NonHCPCluster, Critical, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_machine_pool.go:50
• [0.000 seconds]

Edit machinepool can create spot machinepool - [id: 43251] [day2, feature-machinepool, NonHCPCluster, High, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_machine_pool.go:113
• [0.000 seconds]

Edit User can grant/list/revoke users - [id:36128] [day2, feature-user, Critical, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_user.go:40
• [0.000 seconds]

Edit nodepool can create/edit/list/delete nodepool - [id:56782] [day2, feature-nodepool, NonClassicCluster, Critical, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_node_pool.go:73
• [0.000 seconds]

Edit nodepool can create nodepool with defined subnets - [id:60202] [day2, feature-nodepool, NonClassicCluster, Critical, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_node_pool.go:182
• [0.000 seconds]

Edit nodepool can create nodepool with tuning config - [id:63178] [day2, feature-nodepool, NonClassicCluster, Critical, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_node_pool.go:304
• [0.000 seconds]

Edit nodepool does support 'version' parameter on nodepool - [id:61138] [day2, feature-nodepool, NonClassicCluster, High, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_node_pool.go:374
• [0.000 seconds]

Edit nodepool can validate the version parameter on nodepool creation/editing - [id:61139] [day2, feature-nodepool, NonClassicCluster, Medium, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_node_pool.go:476
• [0.000 seconds]

Edit nodepool can list/describe/delete nodepool upgrade policies - [id:67414] [day2, feature-nodepool, NonClassicCluster, Critical, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_node_pool.go:570
• [0.000 seconds]

Ran 37 of 38 Specs in 0.001 seconds
SUCCESS! -- 37 Passed | 0 Failed | 0 Pending | 1 Skipped
PASS
</details>

<details>
<summary>$ ginkgo run -v --label-filter 'Exclude' --dry-run tests/e2e/</summary>
Will run 37 of 38 specs

[BeforeSuite] 
${ROSA_PATH}/tests/e2e/e2e_suite_test.go:19
[BeforeSuite] PASSED [0.000 seconds]

S

Edit kubeletconfig can create podPidLimit via rosacli will work well - [id:68828] [day2, feature-kubeletconfig, Critical, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_kubelet_config.go:47
• [0.000 seconds]

Edit kubeletconfig can update podPidLimit via rosacli will work well - [id:68835] [day2, feature-kubeletconfig, Critical, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_kubelet_config.go:88
• [0.000 seconds]

Edit kubeletconfig can delete podPidLimit via rosacli will work well - [id:68836] [day2, feature-kubeletconfig, Critical, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_kubelet_config.go:142
• [0.000 seconds]

Validate user try to create cluster with invalid usernames, passwords or unsupported configurations - [id:66362] [day2, feature-user, Critical, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_user.go:144
• [0.000 seconds]

Edit IAM can create/list/delete account-roles - [id:43070] [day2, feature-roles, High, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_iam_roles.go:62
• [0.000 seconds]

Edit IAM can create operator-roles prior to cluster creation - [id:60971] [day2, feature-roles, High, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_iam_roles.go:239
• [0.000 seconds]

Edit IAM can validate when user create operator-roles to cluster - [id:43051] [day2, feature-roles, High, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_iam_roles.go:511
• [0.000 seconds]

Edit IAM can validate that upgrade account-roles with the managed policies should be forbidden - [id:57441] [day2, feature-roles, High, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_iam_roles.go:553
• [0.000 seconds]

Edit IAM can delete account-roles with --hosted-cp and --classic - [id:62083] [day2, feature-roles, High, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_iam_roles.go:615
• [0.000 seconds]

Edit IAM can validate create/link/unlink user-role - [id:52580] [day2, feature-roles, High, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_iam_roles.go:672
• [0.000 seconds]

Edit IAM can create/delete/unlink/link ocm-roles in auto mode - [id:46187] [day2, feature-roles, High, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_iam_roles.go:782
• [0.000 seconds]

Edit IAM can create/link/unlink/delete user-role in auto mode - [id:52419] [day2, feature-roles, High, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_iam_roles.go:918
• [0.000 seconds]

Edit OIDC config can create/list/delete BYO oidc config in auto mode - [id:57570] [day2, feature-oidcconfig, High, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_oidc_config.go:54
• [0.000 seconds]

Verify the creation of rosa cluster with volume size will work - [id:66359] [day1-post, Critical, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_day1_post.go:52
• [0.000 seconds]

Verify the creation of ROSA cluster with default-mp-labels option will succeed - [id:57056] [day1-post, Critical, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_day1_post.go:95
• [0.000 seconds]

Verify the windows certificates expiration - [id:64040] [day1-post, Medium, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_day1_post.go:131
• [0.000 seconds]

Edit nodepool can create/edit/list/delete nodepool - [id:56782] [day2, feature-nodepool, NonClassicCluster, Critical, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_node_pool.go:73
• [0.000 seconds]

Edit nodepool can create nodepool with defined subnets - [id:60202] [day2, feature-nodepool, NonClassicCluster, Critical, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_node_pool.go:182
• [0.000 seconds]

Edit nodepool can create nodepool with tuning config - [id:63178] [day2, feature-nodepool, NonClassicCluster, Critical, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_node_pool.go:304
• [0.000 seconds]

Edit nodepool does support 'version' parameter on nodepool - [id:61138] [day2, feature-nodepool, NonClassicCluster, High, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_node_pool.go:374
• [0.000 seconds]

Edit nodepool can validate the version parameter on nodepool creation/editing - [id:61139] [day2, feature-nodepool, NonClassicCluster, Medium, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_node_pool.go:476
• [0.000 seconds]

Edit nodepool can list/describe/delete nodepool upgrade policies - [id:67414] [day2, feature-nodepool, NonClassicCluster, Critical, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_node_pool.go:570
• [0.000 seconds]

Edit default ingress can update on rosa HCP cluster - [id:63323] [day2, feature-ingress, Critical, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_ingress.go:43
• [0.000 seconds]

Edit User can grant/list/revoke users - [id:36128] [day2, feature-user, Critical, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_user.go:40
• [0.000 seconds]

Edit cluster -  can check the description of the cluster - [id:34102] [day2, feature-cluster, Medium, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_cluster.go:49
• [0.000 seconds]

Edit cluster -  can restrict master API endpoint to direct, private connectivity or not - [id:38850] [day2, feature-cluster, High, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_cluster.go:106
• [0.000 seconds]

Edit cluster -  can disable workload monitoring on/off - [id:45159] [day2, feature-cluster, High, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_cluster.go:168
• [0.000 seconds]

Edit cluster -  can validate for deletion of upgrade policy of rosa cluster - [id:38787] [day2, feature-cluster, Medium, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_cluster.go:216
• [0.000 seconds]

Edit cluster -  can allow sts cluster installation with compatible policies - [id:45161] [day2, feature-cluster, High, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_cluster.go:241
• [0.000 seconds]

Network verifier can verify network - [id:64917] [day2, feature-networkverifier, High, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_network_verifier.go:44
• [0.000 seconds]

Region can list regions - [id:55729] [day2, feature-region, High, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_region.go:35
• [0.000 seconds]

Edit machinepool can create machinepool with volume size set - [id:66872] [day2, feature-machinepool, NonHCPCluster, Critical, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_machine_pool.go:50
• [0.000 seconds]

Edit machinepool can create spot machinepool - [id: 43251] [day2, feature-machinepool, NonHCPCluster, High, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_machine_pool.go:113
• [0.000 seconds]

Edit IDP can create/describe/delete admin user - [id:35878] [day2, feature-idp, Critical, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_idp.go:55
• [0.000 seconds]

Edit IDP can create/List/Delete IDPs for rosa clusters - [id:35896] [day2, feature-idp, Critical, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_idp.go:138
• [0.000 seconds]

Edit IDP can create/delete the HTPasswd IDPs - [id:49137] [day2, feature-idp, Critical, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_idp.go:275
• [0.000 seconds]

OIDC provider validate when user create oidc-provider to cluster - [id:43046] [day2, feature-oidcprovider, High, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_oidc_provider.go:42
• [0.000 seconds]

Ran 37 of 38 Specs in 0.001 seconds
SUCCESS! -- 37 Passed | 0 Failed | 0 Pending | 1 Skipped
</details>

<details>
<summary>$ ginkgo run -v --label-filter 'MigrationToVerify' --dry-run tests/e2e/</summary>
Will run 37 of 38 specs

[BeforeSuite] 
${ROSA_PATH}/tests/e2e/e2e_suite_test.go:19
[BeforeSuite] PASSED [0.000 seconds]

OIDC provider validate when user create oidc-provider to cluster - [id:43046] [day2, feature-oidcprovider, High, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_oidc_provider.go:42
• [0.000 seconds]

Edit IDP can create/describe/delete admin user - [id:35878] [day2, feature-idp, Critical, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_idp.go:55
• [0.000 seconds]

Edit IDP can create/List/Delete IDPs for rosa clusters - [id:35896] [day2, feature-idp, Critical, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_idp.go:138
• [0.000 seconds]

Edit IDP can create/delete the HTPasswd IDPs - [id:49137] [day2, feature-idp, Critical, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_idp.go:275
• [0.000 seconds]

Edit machinepool can create machinepool with volume size set - [id:66872] [day2, feature-machinepool, NonHCPCluster, Critical, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_machine_pool.go:50
• [0.000 seconds]

Edit machinepool can create spot machinepool - [id: 43251] [day2, feature-machinepool, NonHCPCluster, High, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_machine_pool.go:113
• [0.000 seconds]

Region can list regions - [id:55729] [day2, feature-region, High, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_region.go:35
• [0.000 seconds]

Edit IAM can create/list/delete account-roles - [id:43070] [day2, feature-roles, High, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_iam_roles.go:62
• [0.000 seconds]

Edit IAM can create operator-roles prior to cluster creation - [id:60971] [day2, feature-roles, High, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_iam_roles.go:239
• [0.000 seconds]

Edit IAM can validate when user create operator-roles to cluster - [id:43051] [day2, feature-roles, High, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_iam_roles.go:511
• [0.000 seconds]

Edit IAM can validate that upgrade account-roles with the managed policies should be forbidden - [id:57441] [day2, feature-roles, High, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_iam_roles.go:553
• [0.000 seconds]

Edit IAM can delete account-roles with --hosted-cp and --classic - [id:62083] [day2, feature-roles, High, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_iam_roles.go:615
• [0.000 seconds]

Edit IAM can validate create/link/unlink user-role - [id:52580] [day2, feature-roles, High, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_iam_roles.go:672
• [0.000 seconds]

Edit IAM can create/delete/unlink/link ocm-roles in auto mode - [id:46187] [day2, feature-roles, High, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_iam_roles.go:782
• [0.000 seconds]

Edit IAM can create/link/unlink/delete user-role in auto mode - [id:52419] [day2, feature-roles, High, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_iam_roles.go:918
• [0.000 seconds]

Edit cluster -  can check the description of the cluster - [id:34102] [day2, feature-cluster, Medium, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_cluster.go:49
• [0.000 seconds]

Edit cluster -  can restrict master API endpoint to direct, private connectivity or not - [id:38850] [day2, feature-cluster, High, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_cluster.go:106
• [0.000 seconds]

Edit cluster -  can disable workload monitoring on/off - [id:45159] [day2, feature-cluster, High, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_cluster.go:168
• [0.000 seconds]

Edit cluster -  can validate for deletion of upgrade policy of rosa cluster - [id:38787] [day2, feature-cluster, Medium, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_cluster.go:216
• [0.000 seconds]

Edit cluster -  can allow sts cluster installation with compatible policies - [id:45161] [day2, feature-cluster, High, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_cluster.go:241
• [0.000 seconds]

Verify the creation of rosa cluster with volume size will work - [id:66359] [day1-post, Critical, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_day1_post.go:52
• [0.000 seconds]

Verify the creation of ROSA cluster with default-mp-labels option will succeed - [id:57056] [day1-post, Critical, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_day1_post.go:95
• [0.000 seconds]

Verify the windows certificates expiration - [id:64040] [day1-post, Medium, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_day1_post.go:131
• [0.000 seconds]

Edit OIDC config can create/list/delete BYO oidc config in auto mode - [id:57570] [day2, feature-oidcconfig, High, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_oidc_config.go:54
• [0.000 seconds]

Edit kubeletconfig can create podPidLimit via rosacli will work well - [id:68828] [day2, feature-kubeletconfig, Critical, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_kubelet_config.go:47
• [0.000 seconds]

Edit kubeletconfig can update podPidLimit via rosacli will work well - [id:68835] [day2, feature-kubeletconfig, Critical, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_kubelet_config.go:88
• [0.000 seconds]

Edit kubeletconfig can delete podPidLimit via rosacli will work well - [id:68836] [day2, feature-kubeletconfig, Critical, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_kubelet_config.go:142
• [0.000 seconds]

Edit User can grant/list/revoke users - [id:36128] [day2, feature-user, Critical, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_user.go:40
• [0.000 seconds]

Validate user try to create cluster with invalid usernames, passwords or unsupported configurations - [id:66362] [day2, feature-user, Critical, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_user.go:144
• [0.000 seconds]

S

Network verifier can verify network - [id:64917] [day2, feature-networkverifier, High, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_network_verifier.go:44
• [0.000 seconds]

Edit nodepool can create/edit/list/delete nodepool - [id:56782] [day2, feature-nodepool, NonClassicCluster, Critical, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_node_pool.go:73
• [0.000 seconds]

Edit nodepool can create nodepool with defined subnets - [id:60202] [day2, feature-nodepool, NonClassicCluster, Critical, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_node_pool.go:182
• [0.000 seconds]

Edit nodepool can create nodepool with tuning config - [id:63178] [day2, feature-nodepool, NonClassicCluster, Critical, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_node_pool.go:304
• [0.000 seconds]

Edit nodepool does support 'version' parameter on nodepool - [id:61138] [day2, feature-nodepool, NonClassicCluster, High, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_node_pool.go:374
• [0.000 seconds]

Edit nodepool can validate the version parameter on nodepool creation/editing - [id:61139] [day2, feature-nodepool, NonClassicCluster, Medium, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_node_pool.go:476
• [0.000 seconds]

Edit nodepool can list/describe/delete nodepool upgrade policies - [id:67414] [day2, feature-nodepool, NonClassicCluster, Critical, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_node_pool.go:570
• [0.000 seconds]

Edit default ingress can update on rosa HCP cluster - [id:63323] [day2, feature-ingress, Critical, MigrationToVerify, Exclude]
${ROSA_PATH}/tests/e2e/test_rosacli_ingress.go:43
• [0.000 seconds]

Ran 37 of 38 Specs in 0.001 seconds
SUCCESS! -- 37 Passed | 0 Failed | 0 Pending | 1 Skipped

</details>

### CODE REVIEW

File: tests/e2e/test_rosacli_cluster.go [hunk 1]
```diff
@@ -0,0 +1,307 @@
...
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	labels "github.com/openshift/rosa/tests/ci/labels"
+	config "github.com/openshift/rosa/tests/utils/config"
+	rosacli "github.com/openshift/rosa/tests/utils/exec/rosacli"
+)
+
+var _ = Describe("Edit cluster - ",
```
Comment (yasun1) [inline tests/e2e/test_rosacli_cluster.go:16]: As the case title will be combined with `Describe` and `It`, do we need `-` here?

File: tests/e2e/test_rosacli_cluster.go [hunk 2]
```diff
@@ -0,0 +1,307 @@
...
+	"fmt"
+	"strconv"
+	"strings"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	labels "github.com/openshift/rosa/tests/ci/labels"
+	config "github.com/openshift/rosa/tests/utils/config"
+	rosacli "github.com/openshift/rosa/tests/utils/exec/rosacli"
```
Comment (yasun1) [inline tests/e2e/test_rosacli_cluster.go:13]: If the alias is as same as the package, I do not think we need to define the alias.

File: tests/e2e/test_rosacli_cluster.go (hunk 1 above)
Comment (radtriste) [reply tests/e2e/test_rosacli_cluster.go:16]: removed

File: tests/e2e/test_rosacli_cluster.go (hunk 2 above)
Comment (radtriste) [reply tests/e2e/test_rosacli_cluster.go:13]: yeah I just copied the code from the other repo. Will make a pass to harmonize that. Good point

Comment (radtriste) [reply tests/e2e/test_rosacli_cluster.go]: updated

**Review by robpblake — APPROVED** [review]
/lgtm

**Pull request merged**